### ISSUE TITLE
UTF-8 issue with 0.34.0 (?)

### ISSUE THREAD

**Issue by pghalliday** [bbatsov/rubocop#2226]
I was happily running 0.33.0 but this morning I upgraded to 0.34.0 and now get the following error

```
Running RuboCop...
Inspecting 30 files


0 files inspected, no offenses detected
"\xFD" from ASCII-8BIT to UTF-8
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:33:in `write'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:33:in `block in save'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:29:in `open'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/result_cache.rb:29:in `save'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:89:in `process_file'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:57:in `block in inspect_files'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:55:in `each'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:55:in `inspect_files'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/runner.rb:33:in `run'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/cli.rb:26:in `run'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:50:in `run_cli'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:38:in `run_main_task'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:30:in `block (2 levels) in initialize'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/file_utils_ext.rb:58:in `verbose'
/var/lib/gems/2.1.0/gems/rubocop-0.34.0/lib/rubocop/rake_task.rb:26:in `block in initialize'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:240:in `call'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:240:in `block in execute'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:235:in `each'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:235:in `execute'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'
/usr/lib/ruby/2.1.0/monitor.rb:211:in `mon_synchronize'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:201:in `block in invoke_prerequisites'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:199:in `each'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:199:in `invoke_prerequisites'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:178:in `block in invoke_with_call_chain'
/usr/lib/ruby/2.1.0/monitor.rb:211:in `mon_synchronize'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:172:in `invoke_with_call_chain'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/task.rb:165:in `invoke'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:150:in `invoke_task'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:106:in `each'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:106:in `block in top_level'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:115:in `run_with_threads'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:100:in `top_level'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:78:in `block in run'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:176:in `standard_exception_handling'
/var/lib/gems/2.1.0/gems/rake-10.4.2/lib/rake/application.rb:75:in `run'
/var/lib/gems/2.1.0/gems/rake-10.4.2/bin/rake:33:in `<top (required)>'
/usr/local/bin/rake:23:in `load'
/usr/local/bin/rake:23:in `<main>'
```

After switching back to 0.33.0 everything passed again

**Comment by jonas054** [comment]
Duplicate of #2213, fixed in #2221. So, fixed on `master` but not released as a gem version yet.

**Comment by pghalliday** [comment]
ok thanks
